extras/jackson-support/src/main/java/com/palantir/remoting3/ext/jackson/ObjectMappers.java

```diff
@@ -116,6 +117,7 @@ public static ObjectMapper withDefaultModules(ObjectMapper mapper) {
                 .disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE)
                 .disable(DeserializationFeature.WRAP_EXCEPTIONS)
                 .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
+                .disable(MapperFeature.ALLOW_COERCION_OF_SCALARS)
```

Comment:
One thing to note is that they've moved this from `MapperFeature` -> `DeserializationFeature` to be released in 3.X, so this will be a guaranteed break when we eventually adopt jackson 3.X.

https://github.com/FasterXML/jackson-databind/blob/481592ca8386bb4e9ce79bfc0596bb7cf85ee780/src/main/java/com/fasterxml/jackson/databind/DeserializationFeature.java#L334

Reply:
How so? This would only be a dev break in http when picking up 3.X, no? Since it is a major version for jackson, there will be other breaking changes regardless.

Reply:
given that DeserializationFeature does exist, might as well use? Minimizing our breaks always good

Reply:
ideally you have something that works with both, because right now it's the only breaking change I can see that would affect remoting

Reply:
https://github.com/FasterXML/jackson/wiki/Jackson-Release-3.0

Reply:
DeserializationFeature doesn't have this feature until 3.0

Reply:
ah, ok